config/kubernetes/org.yaml

```diff
@@ -1915,29 +1915,35 @@ teams:
     description: ""
     maintainers:
     - apelisse
+    - dims
     members:
     - mengqiy
     - monopole
     - pwittrock
     - thockin
+    - dims
     privacy: closed
   utils-maintainers:
     description: ""
     maintainers:
     - apelisse
+    - dims
     members:
     - mengqiy
     - monopole
     - pwittrock
     - thockin
+    - dims
```

Comment:
If you add yourself as a maintainer, you don't need to add as a member (peribolos will throw an error)